connectors/twitter/src/main/scala/com/raphtory/twitter/spout/LiveTwitterSpout.scala

```diff
@@ -33,7 +33,7 @@ class LiveTwitterSpout() extends Spout[Tweet] {
   val tweetQueue = new ConcurrentLinkedQueue[Tweet]()
   val spout      = new LiveTwitterAddSpout(tweetQueue)
 
-  override def close(): Unit = System.exit(1)
+  override def close(): Unit = ???
```

Comment:
Can this be changed to close the LiveTwitterAddSpout object? This can then be merged in.